massa-bootstrap/src/client.rs

```diff
@@ -117,11 +119,12 @@ async fn stream_final_state(
                 BootstrapServerMessage::SlotTooOld => {
                     info!("Slot is too old retry bootstrap from scratch");
                     *next_bootstrap_message = BootstrapClientMessage::AskFinalStatePart {
+                        last_slot: None,
                         last_key: None,
-                        slot: None,
                         last_async_message_id: None,
-                        last_cycle: None,
+                        last_cycle_step: PoSCycleStreamingStep::Started,
```

Comment:
If the slot is too old, we should completely reboot the node and restart the bootstrap from scratch with another node (because here we will retry from the same node)